### ISSUE TITLE
[l10n] copy files between settings and l10n repo

### ISSUE THREAD

**Issue by data-sync-user** [mozilla/fxa#7319]
We need copy the en-US file into the l10n repo and copy the translated files into settings.

See https://jira.mozilla.com/browse/FXA-2344?focusedCommentId=216287&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-216287 and https://jira.mozilla.com/browse/FXA-2344?focusedCommentId=216294&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-216294 ([https://github.com/mozilla/fxa/issues/6036#issuecomment-760301898](https://github.com/mozilla/fxa/issues/6036#issuecomment-760301898) and https://github.com/mozilla/fxa/issues/6036#issuecomment-760306388).

┆Issue is synchronized with this [Jira Task](https://jira.mozilla.com/browse/FXA-2934)
┆Issue Number: FXA-2934

**Comment by data-sync-user** [comment]
➤ Barry Chen commented:

https://github.com/mozilla/fxa-content-server-l10n/pull/464 ( https://github.com/mozilla/fxa-content-server-l10n/pull/464 ) is the patch for copying from settings into the l10n repo.

**Comment by data-sync-user** [comment]
➤ Barry Chen commented:

I will do FXA-2156 as a part of this task.

**Comment by data-sync-user** [comment]
➤ Barry Chen commented:

Put this back to To Do while we get the l10n side of things done.

**Comment by data-sync-user** [comment]
➤ Barry Chen commented:

Bumped the points up because the number of changes needed for https://github.com/mozilla/fxa-content-server-l10n/pull/466

**Comment by data-sync-user** [comment]
➤ Barry Chen commented:

I've been going through the comments on https://github.com/mozilla/fxa-content-server-l10n/pull/466/files ( https://github.com/mozilla/fxa-content-server-l10n/pull/466/files ) and making changes in FxA based on that feedback.  The WIP branch is at https://github.com/chenba/fxa/tree/2934-ftl-fixes-II. ( https://github.com/chenba/fxa/tree/2934-ftl-fixes-II. )  I mark each comment as resolved when I make the corresponding change in FxA, so the unresolved comments is a to-do list.

Keep in mind that settings.ftl in FxA is a generated file, so ensure that the changes go into the smaller individual files.  If you don't have the settings service running while making edits, then you need to build settings to generate a new settings.ftl before committing.  (Depending on the fix, you might need to update the code where the strings are used.)

Once everything's fixed, get the patch merged into FxA, and then copy the new settings.ftl to the l10n PR to get another review.

**Comment by data-sync-user** [comment]
➤ Jared Hirsch commented:

We have decided not to try porting over existing strings, as there weren't many matches.

We've landed the patch into FxA in https://github.com/mozilla/fxa/pull/7540. ( https://github.com/mozilla/fxa/pull/7540. )

The l10n PR https://github.com/mozilla/fxa-content-server-l10n/pull/466 ( https://github.com/mozilla/fxa-content-server-l10n/pull/466 ) has been updated with the latest settings.ftl file. Now we're waiting on a review there from l10n folks.

**Comment by data-sync-user** [comment]
➤ Jared Hirsch commented:

We've now landed the initial en_US ftl strings in the l10n repo. I think that's all that was needed here; closing.
